MathlibTest/delaborators.lean

```diff
@@ -100,6 +122,17 @@ open PiNotation
 #guard_msgs in
 #check (x : Nat) → P x
 
+/-!
+`Π` notation doesn't have binder predicates.
```

Comment:
```suggestion
`Π` notation doesn't have binder predicates yet.
```

why not? is this a future TODO or something else? Maybe expand the sentence to make it more instructive.